SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -206,7 +206,10 @@ void InterestFactory::AddServerSelfInterest(Interest& OutInterest, const Worker_
 	// Add a query for components all servers need to read client data
 	Query ClientQuery;
 	ClientQuery.Constraint.EntityIdConstraint = EntityId;
-	ClientQuery.ResultComponentIds = ServerAuthInterestResultType;
+	// Temp fix for invalid initial auth server checkout constraints - UNR-3683
+	// Using full snapshot ensure all components are available on checkout. Remove when root issue is resolved.
+	//ClientQuery.ResultComponentIds = ServerAuthInterestResultType;
```

Comment:
nit: remove line, ticket should be enough to be able to fix this without leaving commented out code here